frontend/packages/dev-console/src/components/helm/details/history/HelmReleaseHistoryRow.tsx

```diff
@@ -2,10 +2,54 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import { Status } from '@console/shared';
 import { TableRow, TableData, RowFunction } from '@console/internal/components/factory';
-import { Timestamp } from '@console/internal/components/utils';
+import { Timestamp, Kebab } from '@console/internal/components/utils';
+import { confirmModal } from '@console/internal/components/modals';
+import { coFetchJSON } from '@console/internal/co-fetch';
 import { tableColumnClasses } from './HelmReleaseHistoryHeader';
 
+const confirmModalRollbackHelmRelease = (
+  releaseName: string,
+  namespace: string,
+  revision: number,
+) => {
+  const message = (
+    <>
+      Are you sure you want to rollback <strong>{releaseName}</strong> to{' '}
+      <strong>Revision {revision}</strong>
+    </>
+  );
+
+  const payload = {
+    namespace,
+    name: releaseName,
+    version: revision,
+  };
+
+  const executeFn = async () => {
+    try {
+      await coFetchJSON.put('/api/helm/release', payload);
+    } catch (err) {
+      return Promise.reject(err);
+    }
+    window.location.reload();
```

Comment:
Is there nothing else that can trigger this helm release to refresh? Reloading the page is the last thing we want to do.

Reply:
Fixed it.